.travis/build_package.sh

```diff
@@ -39,6 +33,11 @@ function build_demo {
     EXTRA_CXX_FLAGS="-Werror=inconsistent-missing-override"
   fi
   if [ $NEED_3D = 1 ]; then
+    if [ "$TRAVIS_OS_NAME" = "osx" ]; then
+  	  cmake -DCGAL_DIR="$ROOT/build" -DQGLVIEWER_INCLUDE_DIR="$ROOT/$DEMO/qglviewer/QGLViewer/QGLViewer.framework/Headers" -DQGLVIEWER_LIBRARIES="$ROOT/$DEMO/qglviewer/QGLViewer/QGLViewer.framework" -DCMAKE_CXX_FLAGS_RELEASE="-DCGAL_NDEBUG" ..
+    else
+   	  cmake -DCGAL_DIR="$ROOT/build" -DQGLVIEWER_INCLUDE_DIR="$ROOT/$DEMO/qglviewer" -DQGLVIEWER_LIBRARIES="$ROOT/$DEMO/qglviewer/QGLViewer/libQGLViewer.so" -DCMAKE_CXX_FLAGS_RELEASE="-DCGAL_NDEBUG" ..
+    fi
```

Comment:
For information, the MacOS test machine as GeometryFactory uses environment variables. Here is the shell script that is used:
```shell
Qt5_DIR=/Users/cgaltester/Qt/5.5/clang_64/lib/cmake/Qt5
export Qt5_DIR
QGLVIEWERROOT=/Users/cgaltester/libQGLViewer-2.6.3/QGLViewer
export QGLVIEWERROOT

DYLD_FRAMEWORK_PATH=$QGLVIEWERROOT:$DYLD_FRAMEWORK_PATH
export DYLD_FRAMEWORK_PATH
```

it seems both `QGLVIEWERROOT` and `DYLD_FRAMEWORK_PATH` are important.